addons/l10n_us/models/res_bank.py

```diff
@@ -0,0 +1,19 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError
+
+
+class ResBank(models.Model):
+    _inherit = "res.bank"
+
+    intermediary_bank_id = fields.Many2one(
+        "res.bank",
+        domain="[('id', '!=', id)]",
+        help="An intermediary bank facilitates international wire transfers between your bank and the beneficiary's bank when they don’t have a direct relationship.",
```

Comment:
I would still export the pot since we are on a localisation they are not automatically exported. Usually when working on localisation in stable and in master we export pot and make the po since here it's already in english the po is not useful :smile:

Reply:
[I responded below](https://github.com/odoo/odoo/pull/183437#issuecomment-2560293191).